### PR TITLE
Development data and sql queries that will create the table and populate with demo data

### PR DESCRIPTION
**Pull request by debasishbsws**

### CODE REVIEW

**Comment by debasishbsws** [comment]
@Flanker-shyam sql queries check karna

File: dev-database.sql [hunk 1]
```diff
@@ -48,4 +50,37 @@ ALTER TABLE `institute_course` ADD FOREIGN KEY (`course_id`) REFERENCES `course`
...
+
+
+-- Populate `question_paper` table
+INSERT INTO `question_paper` (`url`, `year`, `semester`, `exam_type`, `title`, `institute_id`, `course_id`)
+VALUES ('Question Paper URL 1', '2022', 'Spring', 'ENDSEM', 'DEMO Question Paper Title 1', 1, 1),
+       ('Question Paper URL 2', '2022', 'Spring', 'MIDSEM', 'DEMO Question Paper Title 1', 1, 1),
+       ('Question Paper URL 2', '2022', 'Spring', 'MIDSEM', 'DEMO Question Paper Title 2', 1, 1),
+       ('Question Paper URL 3', '2022', 'Autumn', 'ENDSEM', 'DEMO Question Paper Title 3', 1, 1),
+       ('Question Paper URL 1', '2021', 'Autumn', 'ENDSEM', 'DEMO Question Paper Title 1', 1, 1),
+       ('Question Paper URL 4', '2021', 'Spring', 'ENDSEM', 'DEMO Question Paper Title 4', 1, 2);
```
Comment (debasishbsws) [inline dev-database.sql:86]: proper link should be added after merge

**Review by Flanker-shyam — COMMENTED** [review]
taking too many details is redundant, why we need street address??

and also how to gonna handle google map thing?? it's not clear, I guess it is redundant too.

**Comment by debasishbsws** [comment]
> taking too many details is redundant, why we need street address??
> 
> and also how to gonna handle google map thing?? it's not clear, I guess it is redundant too.

Yup we may not need the street and pin etc feel free to add suggestions in the comment.

And for the Google map thing, we can use the Google Maps Geocoding API. 
an example code that uses place id to get an address. (AI generated) may need some changes

```
const axios = require('axios');

async function getAddressDetails(googleMapsLink) {
  try {
    // Extract the place ID from the Google Maps link
    const [, query] = googleMapsLink.split('/maps/');
    const placeId = query.split('?')[0];

    // Make a request to the Geocoding API
    const response = await axios.get('https://maps.googleapis.com/maps/api/geocode/json', {
      params: {
        place_id: placeId,
        key: 'YOUR_API_KEY' // Replace with your Google Maps API key
      }
    });

    // Extract the formatted address from the API response
    const { results } = response.data;
    if (results.length > 0) {
      const formattedAddress = results[0].formatted_address;
      return formattedAddress;
    } else {
      throw new Error('No address found');
    }
  } catch (error) {
    console.error('Error:', error.message);
  }
}

// Example usage
const googleMapsLink = 'https://goo.gl/maps/TxPD2nAYrVkFSAfc9';
getAddressDetails(googleMapsLink)
  .then(address => {
    console.log('Address:', address);
  })
  .catch(error => {
    console.error('Error:', error.message);
  });
```

**Comment by Flanker-shyam** [comment]
seems fine, Yepp we can integrate that, but that the similar ques, what is the purpose of taking map location, In context to our application?? 
taking state and country should be sufficient for us to differentiate between institutes.

**Comment by Flanker-shyam** [comment]
That too, user can take enter comfortably.

**Comment by debasishbsws** [comment]
Okay, I will not be needed for our application IG. So we do not need the Address table and just take the country, state info and save it to the institute table it self.

**Comment by Flanker-shyam** [comment]
Yes that would be fine !

**Comment by debasishbsws** [comment]
Added the docker compose file that will create the MySQL server as a docker container and run the `dev-database.sql` to create the demo schema and fill it with demo data. also run a `phpmyadmin` server on `localhost:8080` we can use to connect to the database.

**Review by Flanker-shyam — APPROVED** [review]
All good !!

**Pull request merged**